python/setup.py

```diff
@@ -41,8 +41,6 @@
         'License :: OSI Approved :: BSD License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```

Comment:
Do we support 3.7, 3.8 and 3.9?

Reply:
I've been running with 3.9, I'll add those versions.